Assignment_1/src/service/TaxService.java

```diff
@@ -0,0 +1,40 @@
+package service;
+
+//TODO: import exception and fix exception errors
+import model.Item;
+
+public class TaxService{
+    
+    public Item calculateTax(Item item) throws InvalidItemTypeException {
+        String type = item.getType();
+        double tax;
+        switch(type) {
+            case "raw":
+                tax = item.getPrice()*(12.5/100);
+                item.setTax(tax);
+                break;
+            case "manufactured":
+                tax = item.getPrice()*12.5/100;
+                tax += (item.getPrice() + tax)*2/100;
```

Comment:
Need to create proper methods to do calculations (and avoid short hand operators)

Reply:
Removed short hand operators and created methods for calculating tax